tests/admin_scripts/tests.py

```diff
@@ -2597,6 +2599,11 @@ def test_file_without_extension(self):
                     fh.read().strip(), "# some file for customtestproject test project"
                 )
 
+    @unittest.skipIf(
+        sys.platform == "win32",
+        "Black triggers unicode error parsing the custom template on Windows.",
+    )
```

Comment:
Needing to skip this on Windows for now. 

Black is struggling with the custom project template (only on Windows) and I'm not yet sure why. 

```
error:` cannot format C:\Users\carlt\AppData\Local\Temp\django_bkwhxley\tmp3vzukz0u\test_project\project_dir\manage.py: cannot use --safe with this file; failed to parse source file AST: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape (<unknown>, line 5)
This could be caused by running Black with an older Python version that does not support new syntax used in your source file.
```

Reply:
I'd _guess_ it's the `C:\Users` part being incorrectly escaped:
```
In [1]: print("C:\Users")
print("C:\Users")
                ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape
In [2]: print(r"C:\Users")
C:\Users
```

Reply:
@kezabelle — Super! That looks quite plausible. 🎁 

It's feeding time at the zoo here now… — I shall have a look in the morning, if nobody get's there first (😜)

Reply:
This escaping issue is cleverer than I'm able to dig down to. 

* Calling black directly AFAICS seems to work correctly. 
* The arguments reach the black process correctly -- double escaping before callling subprocess leads to various file not found issues. 
* It turns out it works with `--fast` see https://github.com/django/django/pull/15412/commits/c74b6a754a302b9b8c7ef33f2ad3b7847bf7645a so I think it's internal to black. (Need to reduce still but...)

Gonna pause on this, to get the test adjustments finished, but any thoughts welcome.